timesketch/lib/utils.py

```diff
@@ -40,10 +42,11 @@ def read_and_validate_csv(path):
     """
     # Columns that must be present in the CSV file
     mandatory_fields = [
-        u'message', u'timestamp', u'datetime', u'timestamp_desc']
+        u'message', u'datetime', u'timestamp_desc']
 
     with open(path, u'rb') as fh:
-        reader = csv.DictReader(fh)
+
+        reader = csv.DictReader((x.replace('\x00', '') for x in fh))
```

Comment:
Why remove null bytes here instead of having the user provide clean data? Bail out and have the user clean up the input before maybe.